oak_functions/examples/weather_lookup/README.md

```diff
@@ -57,6 +57,42 @@ radius of 40km. See https://s2geometry.io/devguide/examples/coverings for more
 information on coverings. Cell coverings for areas can be visualised using
 https://s2.sidewalklabs.com/regioncoverer/.
 
+### File format
+
+The weather file format looks as follows:
+
+Weather file is a Protobuf compiled file that consists of a list of the
```

Comment:
I would suggest something like:

The weather lookup data file is a [stream of length-delimited binary encoded protobuf entries](https://developers.google.com/protocol-buffers/docs/techniques#streaming). The encoding length of each entry is encoded as a [base 128 varint](https://developers.google.com/protocol-buffers/docs/encoding#varints). Each entry is based on the following proto definition: